R/processing.R

```diff
@@ -79,14 +79,15 @@ complete_cumulative_columns <- function(data) {
     "deaths_total", "cases_total", "recovered_total",
     "hosp_total", "tested_total"
   )
-  for (cumulative_col_name in cumulative_col_names) {
-    if (cumulative_col_name %in% colnames(data)) {
-      data <- fill(data, all_of(cumulative_col_name))
-    }
-  }
+  idx <- cumulative_col_names %in% colnames(data)
+  targets <- cumulative_col_names[idx]
+  map(targets, ~ {
```

Comment:
Sometimes global assignment makes me worried. This might be a goods use of `reduce`? I had a little think and nothing really comes to mind so perhaps this is the optimal solution.

Reply:
Yeah I tend to avoid them where I can, but for this it was the only way I could get it work (although almost certain there will be another way)

Reply:
yes fair enough!